### PR TITLE
retain collection id on query change

### LINKED ISSUE #193: Param inputs do not retain the value when switching between method
Go to https://cennznet.io/#/toolbox (on Rata) and select `nft` module then select `getCollectionInfo` method with the Collection Id as `31`, executing it will send this request over ws

```
{"id":32,"jsonrpc":"2.0","method":"nft_getCollectionInfo","params":[31]}
```

after that, switching the method to `getTokenInfo`, notice the Collection Id is still display `31` but when executing it, the request is sent with params as `[0,0,0]`

```
{"id":36,"jsonrpc":"2.0","method":"nft_getTokenInfo","params":[0,0,0]}
```

### PR DESCRIPTION
**Pull request by aidan-starke**
### Details
 - closes #193
 - save `collectionId` to state
 - if params array includes `CollectionId` set value to stored `collectionId`

### Notes
 - need to merge #194 first

### CODE REVIEW

File: packages/app-toolbox/src/Rpc/Selection.tsx [hunk 1]
```diff
@@ -139,7 +79,7 @@ class Selection extends TxComponent<Props, State> {
         const { rpc = prevState.rpc, accountId = prevState.accountId, values = prevState.values } = newState;
         const reqCount = rpc.params.reduce((count, { isOptional }): number => count + (isOptional ? 0 : 1), 0);
         const isValid = values.reduce((isValid, value): boolean => {
-          return isValid && value.isValid === true;
+          return isValid && value.isValid;
```
Comment (ken-centrality) [inline packages/app-toolbox/src/Rpc/Selection.tsx:82]: why are we removing strict comparison here?

File: packages/react-params/src/index.tsx [hunk 2]
```diff
@@ -91,12 +93,20 @@ class Params extends React.PureComponent<Props, State> {
...
     }
 
+    const collectionIdParam = params.find(({ name }) => name === 'CollectionId');
+
+    if (!!collectionId && !!collectionIdParam) {
+      const index = params.indexOf(collectionIdParam);
+
+      values[index].value = new BN(collectionId);
+    }
+
```
Comment (ken-centrality) [inline packages/react-params/src/index.tsx:109]: Feel like we are only fixing `collectionId`. Are other params input work as expected?

**Review by ken-centrality — COMMENTED** [review]
seems fine, but we should widen the fix to other params rather than just `collectionId` if applicable

File: packages/app-toolbox/src/Rpc/Selection.tsx (hunk 1 above)
Comment (aidan-starke) [reply packages/app-toolbox/src/Rpc/Selection.tsx:82]: webstorm tells me it's redundant haha

**Pull request closed without merging**